tests/Doctrine/Tests/ODM/PHPCR/DocumentManagerTest.php

```diff
@@ -54,7 +54,7 @@ public function testFindTranslation()
      */
     public function testNewInstanceFromConfiguration()
     {
-        $session = $this->getMock('PHPCR\SessionInterface');
+        $session = $this->getMockBuilder('PHPCR\SessionInterface')->getMock();
```

Comment:
The problem is that `createMock` does not exist (even in PHPUnit 5.0), and we are using, by necessity, PHPUnit 4.x in the tests with PHP 5.4